projects/addon-mobile/components/swipe-action/swipe-actions.style.less

```diff
@@ -6,6 +6,13 @@
     --tui-action-gap: 24;
     --tui-actions-padding: 0.5rem;
     --tui-item-size: 44;
+    --t-x: 100%;
+    --t-rtl: 1;
+
+    [dir='rtl'] & {
+        --t-x: 0%;
+        --t-rtl: -1;
```

Comment:
We already have global `--tui-inline` that does that.

Reply:
I used `--t-inline` until this pr merged https://github.com/taiga-family/taiga-ui/pull/11300